lib/eyeshade/base_balance.rb

```diff
@@ -0,0 +1,51 @@
+module Eyeshade
+  class BaseBalance
+    attr_reader :rates,
+                :default_currency,
+                :amount_probi,
+                :amount_bat,
+                :amount_default_currency,
+                :fees_probi,
+                :fees_bat,
+                :fees_default_currency
+
+    # Account types
+    CHANNEL = "channel".freeze
+    OWNER = "owner".freeze
+
+    def initialize(rates, default_currency)
+      @rates = rates
+      @default_currency = default_currency || "BAT"
```

Comment:
In the `publisher_converted_overall_balance` we check to see if the default_currency is blank. Since this is defaulting to be BAT then we can remove that check ✔️

Reply:
I've removed defaulting to BAT since some users are unable withdraw cryptos because of regional restrictions.  Uphold does not have a [bitlicense](https://en.wikipedia.org/wiki/BitLicense).